model_bakery/utils.py

```diff
@@ -2,7 +2,7 @@
 import datetime
 import itertools
 
-from .timezone import tz_aware
+from .timezone import now
```

Comment:
This import is breaking the tests because the `model_bakery.timezone` module directly imports `django`. This is a problem because we do import `model_baker` in [our setup.py](https://github.com/model-bakers/model_bakery/blob/master/setup.py#L4) and the model `model_bakery.utils` is being reference in `model_bakery/__init__.py`. So, everything breaks when running `pip install model_bakery`. 

I know this is ugly, but one way to fix this is to move this import and to place it inside the `seq` code. By doing so, Django will be imported during execution time and so the `setup.py` will work once again.